extensions/kubernetes-config/runtime/src/test/java/io/quarkus/kubernetes/client/runtime/KubernetesConfigSourceProviderTest.java

```diff
@@ -13,10 +14,7 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
-import io.fabric8.kubernetes.api.model.ConfigMap;
-import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
-import io.fabric8.kubernetes.api.model.ConfigMapList;
-import io.fabric8.kubernetes.api.model.DoneableConfigMap;
+import io.fabric8.kubernetes.api.model.*;
```

Comment:
Minor nitpick: Please avoid using start imports

Reply:
Fixed it